src/NUnitEngine/nunit-agent/nunit-agent.csproj

```diff
@@ -2,14 +2,18 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <RootNamespace>nunit.agent</RootNamespace>
-    <TargetFrameworks>net20</TargetFrameworks>
+    <TargetFrameworks>net20;net40</TargetFrameworks>
     <ApplicationManifest>app.manifest</ApplicationManifest>
     <ApplicationIcon>..\..\..\nunit.ico</ApplicationIcon>
     <GenerateSupportedRuntime>false</GenerateSupportedRuntime>
+    <OutputPath>..\..\..\bin\$(Configuration)\agents\</OutputPath>
   </PropertyGroup>
   <ItemGroup Condition="'$(TargetFramework)'=='net20'">
     <Reference Include="System.Runtime.Remoting" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='net40'">
```

Comment:
Same as for src/NUnitEngine/nunit-agent/nunit-agent-x86.csproj

Reply:
Same again.